<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WkfMainWindowUi</class>
 <widget class="QMainWindow" name="WkfMainWindowUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QFrame" name="centralwidget">
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="bannerLabel">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="mMenuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="separator"/>
    <addaction name="mActionExit"/>
   </widget>
   <widget class="QMenu" name="mMenuOptions">
    <property name="title">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QMenu" name="mMenuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <widget class="QMenu" name="mMenuView">
    <property name="title">
     <string>视图</string>
    </property>
    <widget class="QMenu" name="mMenuToolbars">
     <property name="title">
      <string>工具栏</string>
     </property>
    </widget>
    <addaction name="mMenuToolbars"/>
   </widget>
   <widget class="QMenu" name="mMenuTools">
    <property name="title">
     <string>工具</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuDeveloper">
    <property name="title">
     <string>开发者</string>
    </property>
   </widget>
   <addaction name="mMenuFile"/>
   <addaction name="mMenuView"/>
   <addaction name="mMenuOptions"/>
   <addaction name="mMenuTools"/>
   <addaction name="menuDeveloper"/>
   <addaction name="mMenuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="mActionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>mActionExit</sender>
   <signal>triggered()</signal>
   <receiver>WkfMainWindowUi</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
